server/src/lib/api.ts

```diff
@@ -520,4 +523,43 @@ export default class API {
 
     return countNumberResult;
   };
+
+  sendLanguageRequest = async (request: Request, response: Response) => {
+    const { email, languageInfo, languageLocale, reCAPTCHAClientResponse } =
+      request.body;
+
+    if (languageInfo.length === 0 || email.length === 0) {
+      return response
+        .status(500)
+        .send('Incorrect body sent to /language/request');
+    }
+
+    const googleReCAPTCHA = new GoogleReCAPTCHA();
+    const isSuccessfulReCAPTCHA = await googleReCAPTCHA.verify(
+      reCAPTCHAClientResponse
+    );
+
+    if (!isSuccessfulReCAPTCHA) {
+      return response.status(500).send('Incorrect reCAPTCHA');
+    }
```

Comment:
Since reCaptcha doesn't have to do with the actual response of the API (its essentially validation), it might make sense breaking it out into a reusable middleware and have it throw an error to the default error handler instead of handling the validation here?